my-website/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
```

Comment:
Ah I see, you have brought the website into the main code. Nice.

You can ignore my comment above about the .yml file. However, it would still be good to have a .yml file do a basic code compile to check for errors

Reply:
Ok, I'll see if I can check for errors before building.